File: homeworks/hw3/Models/Circle.java
package homeworks.hw3.Models;

public class Circle extends Figure {

    private double radius;

    public Circle(double radius) {
        this.radius = radius;
    }

    public double getPerimeter() {
        return 2 * Math.PI * radius;
    }

    public double getArea() {
        return Math.PI * radius * radius;
    }

    public double getRadius() {
        return radius;
    }

}

File: finaltask/models/circle.py
import math

from finaltask.models.figure import Figure


class Circle(Figure):

    def __init__(self, radius):
        self.radius = radius

    def get_perimeter(self):
        return 2 * math.pi * self.radius

    def get_area(self):
        return math.pi * self.radius * self.radius

    def get_radius(self):
        return self.radius